packages/core-components/report.api.md

```diff
@@ -876,6 +877,9 @@ export const ProxiedSignInPage: (
 export type ProxiedSignInPageProps = SignInPageProps & {
   provider: string;
   headers?: HeadersInit | (() => HeadersInit) | (() => Promise<HeadersInit>);
+  ErrorComponent?: ComponentType<{
+    error?: Error;
```

Comment:
Did this one need to be optional? Looks to me that it'd just leave things open for interpretation (what should the component do when it's not set? Render nothing? Render some generic message?). The error page could have instead generated that generic error. But I guess, then the question of i18n of that message comes up.

Reply:
The default is just the error panel with error message. So I would say this should be optional not to break backwards compatibility